contracts/test/utils/DSTest.sol

```diff
@@ -66,6 +66,13 @@ contract DSTest {
         }
     }
 
+    function assertFalse(bool condition) internal {
```

Comment:
while you're at it, can you add an `assertEq(bool a, bool b)` ? I found it was missing the other day while writing a test for shipyard

```
  function assertEq(bool a, bool b) internal {
        if (a != b) {
            emit log("Error: a == b not satisfied [address]");
            emit log_named_string("  Expected", b ? "true" : "false");
            emit log_named_string("    Actual", a ? "true" : "false");
            fail();
        }
    }
```